tgui/packages/tgui/interfaces/TacticalConsole.js

```diff
@@ -96,6 +110,39 @@ export const TacticalConsole = (props, context) => {
                             average: [0.15, 0.9],
                             bad: [-Infinity, 0.15],
                           }} />
+                        {!!data.additional_weapon_info && ((!!data.modifying_weapon_priorities && !!value.can_modify_priority && (
+                          <Box>
+                            <br />
+                            {"Priority: "}
+                            <NumberInput
+                              value={value.weapon_priority}
+                              width="47px"
+                              minValue={1}
+                              maxValue={900}
+                              onChange={(e, input_number) => act('change_weapon_priority', {
+                                target_weapon: value.weapon_ref,
+                                new_priority: input_number,
+                              })} />
```

Comment:
```suggestion
                              onChange={(e, input_number) => act('change_weapon_priority', {
                                target_weapon: value.weapon_ref,
                                new_priority: input_number
                              })} />
```

Reply:
Doing this is seen as an error by eslint and immediately corrected.